server/tests/controllers/test_status_controller.py

```diff
@@ -61,3 +61,10 @@ async def test_objgraph_400(client, headers, query):
         method="GET", path="/objgraph?" + query, headers=headers, json={},
     )
     assert response.status == 400
+
+
+async def test_healthz_200(client, headers):
```

Comment:
I'll add a test for 425 once [this](https://github.com/athenianco/athenian-api/pull/1337) gets merged since it already has some utilities in place for testing with preloading enabled.